compiler/testData/diagnostics/tests/when/CommaInWhenConditionWithoutArgument.fir.kt

```diff
@@ -7,7 +7,7 @@
  */
 fun foo(x: Int, y: Int): Int =
         when {
-            x > 0, y > 0,<!SYNTAX!>,<!> x < 0 -> 1
+            x > 0<!COMMA_IN_WHEN_CONDITION_WITHOUT_ARGUMENT!>,<!> y > 0,<!SYNTAX!>,<!> x < 0 -> 1
```

Comment:
We need one more report for the second comma?

https://punzki.github.io/Kotlin-DiagnosticTestsDiff/2021-06-29/diff-tests__when__CommaInWhenConditionWithoutArgument.kt.html

Reply:
Ah, I think your intention for `COMMAS` position is to pick all `COMMA` token appearances? Perhaps that's not working as expected?

Reply:
This seems to be a limitation of the test infra (https://github.com/JetBrains/kotlin/blob/a288b8b00e4754a1872b164999c6d3f3b8c8994a/compiler/tests-common-new/tests/org/jetbrains/kotlin/test/frontend/fir/handlers/FirDiagnosticCodeMetaInfo.kt#L25). The error is correctly highlighted inside the IDE. Let me try fixing the test infra to highlight multiple ranges.